config/install/field.storage.node.field_consum.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_consum
+field_name: field_consum
```

Comment:
This field's machine name doesn't match the content model.
![vehicles_field_consumables_machine_name](https://github.com/notS0Solo/swapimigration/assets/1434979/30d7ac1b-ac89-4620-b282-e126f275304c)

Reply:
Fixed